Uno.Gallery/Uno.Gallery.Shared/Entities/SampleCategory.cs

```diff
@@ -15,6 +15,7 @@ public enum SampleCategory
 		Theme,
 		Components,
 		Features,
+		Toolkit,
```

Comment:
Can we have two sections under Components ? 
> Components
    > WinUI controls
    > Toolkit controls